### DISCUSSION TITLE
Accessing llama3

### DISCUSSION THREAD

**Discussion by optimikelabs** [nicobrauchtgit/obsidian-Smart2Brain#104] [Ideas]
Please provide the option to access the llama3 models as they have been available since ollama.
Thank you for considering this suggestion.

![image](https://github.com/your-papa/obsidian-Smart2Brain/assets/31423109/07b6bdf2-edd7-47a5-b4df-f7ee0d4fbe5c)

**Comment by Leo310** [comment]
You should be able to use them already. You can install llama3 through our plugin:

1. Open the command palette
2. Type "Smart Second Brain: Pull Ollama Model"
3. Type the model name you want to install
4. Press install
5. Select the model as any other model in the settings page

Besides that, we will add it to the default/recommended models to make this process easier.
